Dockerfile

```diff
@@ -7,7 +7,7 @@ RUN python -m pip install --upgrade pip
 RUN pip install --no-cache-dir --upgrade -r /code/requirements-prod.txt
 COPY . /code
 
-CMD ["uvicorn", "main:app", "--host", "0.0.0.0", "--port", "8000"]
+CMD ["uvicorn", "main:app", "--host", "0.0.0.0", "--port", "8080"]
```

Comment:
why port change is required?

Reply:
In production sakhi api service is running on 8080 , so we need to change the port  in dockerfile.